extensions/projection/README.md

```diff
@@ -0,0 +1,124 @@
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| proj:epsg        | integer\|null  | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:crs         | string \|null  | Recommended. PROJ string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:extent` fields represent |
+| proj:geometry    | Polygon Object | Recommended. Defines the footprint of this item. |
+| proj:extent      | [number]       | Recommended. Bounding box of the item or asset in the native CRS |
+| proj:centroid    | [number]       | Recommended. Coordinates representing the centroid of the item or asset in the native CRS |
+
+**proj:epsg** - A Coordinate Reference System (CRS) is the native reference system (sometimes called a
+'projection') used by the data, and can usually be referenced using an [EPSG code](http://epsg.io).
+If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
+Points, proj:epsg should be set to null. It should also be set to null if a CRS exists, but for which
+there is no valid EPSG code.
+
+**proj:crs** - A Coordinate Reference System (CRS) is the native reference system (sometimes called a
+'projection') used by the data. This value is a PROJ string.
+If the data does not have a CRS, such as in the case of non-rectified imagery with Ground Control
+Points, proj:crs should be set to null. It should also be set to null if a CRS exists, but for which
+a PROJ string does not exist.
+
+**proj:geometry** - A Polygon object representing the footprint of this item, formatted according the Polygon object format specified in [RFC 7946, sections 3.1.6](https://tools.ietf.org/html/rfc7946), except not necessarily in EPSG:4326 as required by RFC7946.  Specified based on the `proj:crs` field (not necessarily EPSG:4326). Ideally, this will be represented by a Polygon with five coordinates, as the item in the native CRS should be a square aligned to the CRS grid.  It is recommended that either or both of `proj:geometry` and `proj:extent` be defined.
+
+**proj:extent** - Bounding box of the asset represented by this item in the native CRS. Specified as four coordinates based on the CRS defined in the `proj:epsg` and `proj:crs` fields.  First two numbers are coordinates of the lower left corner, followed by coordinates of upper right corner, e.g., \[west, south, east, north]. It is recommended that either or both of `proj:geometry` and `proj:extent` be defined.
```

Comment:
Does saying 

> e.g., [west, south, east, north] 

imply a certain class of CRSs? Perhaps "[xmin, ymin, xmax, ymax]" instead?

Reply:
I think the issue with min/max terminology is that for antimeridian-crossing boxes, that xmin is actually greater than xmax, which doesn't have the same implication with west/east

Reply:
What about a CRS (e.g. [EPSG:22182](http://www.epsg-registry.org/export.htm?wkt=urn:ogc:def:crs:EPSG::22182)) where positive X is "northing"? I'd worry about mistakes being made if coordinates have to be flipped in the expression of extent.

Reply:
Should this align with the bbox definition in the item spec?  In the spec its defined as [lower_left_corner, upper_right_corner].   From my experience,  [xmin,ymin,xmax,ymax] is a modification of the traditional bounding box expression with corner coordinates.  It is more intuitive and almost always correct (with the exception of antimeridian crossings as Phil mentions) but is less clear than the corner expression in my opinion.

Reply:
We should strictly follow the Item and Collection spec, which are following the GeoJSON and WFS3 spec. I think we should not diverge from that.

Reply:
The problem with following the GeoJSON spec (I don't know about WFS3) is that the CRS is assumed to be 4326, so north is positive Y, east is positive X, etc. 

> A GeoJSON object MAY have a member named "bbox" to include  information on the coordinate range for its Geometries, Features, or  FeatureCollections.  The value of the bbox member MUST be an array of  length 2*n where n is the number of dimensions represented in the  contained geometries, with all axes of the most southwesterly point  followed by all axes of the more northeasterly point.  The axes order  of a bbox follows the axes order of geometries.  The "bbox" values define shapes with edges that follow lines of  constant longitude, latitude, and elevation.

At least that's how I'm interpreting the above.... so feel free to correct me.

No such assumption can be made of arbitrary CRSs. You'd have to project back into 4326 (or similar) to fully comply with the spec, completely negating the purpose of this extension spec.

Reply:
I think @geospatial-jeff meant that the extension fields should at least match in name to the GeoJSON terms.  In other words, bbox is used at the GeoJSON feature level to describe the bbox, so if the proj extension provides the bounding box in a different CRS it should follow the same naming convention.
It also avoids confusion with the "extent" field which is used at the collection level and has a quite different meaning (the bbox of the convex hull containing all member Items).

Reply:
I should have named this `bbox` instead of extent if that's used in collection.  I'm also okay leaving this out if it's too ill-defined outside of 4326.